src/app/work-item/work-item-board/work-item-board.component.scss

```diff
@@ -57,6 +57,9 @@
               justify-content: space-between;
             }
           }//boardcard ended
+          .board-card-selected {
+            border: 3px solid #bee1f4;
```

Comment:
You should get the color from PatternFly, using the variables that start with $color-pf-